services/image-rec-master/service/__init__.py

```diff
@@ -8,14 +8,16 @@
 
 class Service:
     def __init__(self, port, imagery_host, imagery_port, interop_host,
-                 interop_port, redis_host, redis_port):
+                 interop_port, redis_host, redis_port, max_auto_targets):
         """Create a new image-rec-master service."""
         self._port = port
 
-        app = create_app()
-        app['redis_url'] = f'redis://{redis_host}:{redis_port}'
-        app['imagery_url'] = f'http://{imagery_host}:{imagery_port}'
-        app['interop_url'] = f'http://{interop_host}:{interop_port}'
+        app = create_app(
+            f'redis://{redis_host}:{redis_port}',
+            f'http://{imagery_host}:{imagery_port}',
+            f'http://{interop_host}:{interop_port}',
+            max_auto_targets
+        )
```

Comment:
You should use keyword arguments, even if the arguments are required, to improve readability.

Reply:
Yeah, will add that tonight.